jake/ossindex/ossindex.py

```diff
@@ -20,21 +20,45 @@ def __init__(self, url='https://ossindex.sonatype.org/api/v3/component-report',
         self._url = url
         self._headers = headers
         self._log = logging.getLogger('jake')
+        self._maxcoords = 128
 
     def get_url(self):
         return self._url
 
     def get_headers(self):
         return self._headers
 
+    def chunk(self, purls):
+        chunks = []
```

Comment:
Like we spoke about, I think this method/function/whatever they call it in Python is fairly complex, and there's likely a more elegant way to do this, I just have no idea what it is. Not saying to change it, just that if someone else comes along and is like "Hey that can be done WAY easier", yeah, I thought you were right :)